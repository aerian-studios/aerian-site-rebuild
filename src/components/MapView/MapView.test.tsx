import * as React from "react";

import * as renderer from "react-test-renderer";

import { MapView } from "./index";

describe("HalfBlock", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <div style={{ height: 400 }}>
                    <MapView position={[51.41942, -2.253581]} />
                </div>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
