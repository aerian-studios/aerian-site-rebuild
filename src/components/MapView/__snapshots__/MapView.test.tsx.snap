// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapView renders correctly 1`] = `
<div
  className=" myClass"
>
  <h2>
    MapView
  </h2>
</div>
`;
